PatternPal/PatternPal.Tests/New_TestCasesRecognizers/Strategy/StrategyTestCase12.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Text;
 
 namespace PatternPal.Tests.TestClasses.Strategy
```

Comment:
This implementation does not correspond to the explanation above.
The concrete strategy does not implement the interface anymore, yet in the explanation, there is a check at "is an implementation of the strategy interface".
Please also check that the other implementations do correspond with their explanation. Just to be sure.